include/openenclave/bits/types.h

```diff
@@ -50,6 +59,11 @@ typedef long long ptrdiff_t;
 typedef long long intptr_t;
 typedef long long time_t;
 typedef long long suseconds_t;
+
+#ifndef __cplusplus
+typedef short wchar_t;
```

Comment:
Not clear why we need this redefinition in the same file.

Reply:
For windows

Reply:
Although it does seem like the windows types aren't really necessary now that I think about it. Special type definitions shouldn't be needed on the host side. I mostly just defined it for parity with the other types which are defined for both.

Reply:
Ah ok. I see the `#elfi defined(_MSC_VER)` above.